docs/docker-setup-guide.md

```diff
@@ -6,28 +6,39 @@ backend into a single image that can be built with just a few commands.
 ## Requirements
 
 - Docker
-- Node.js (if running the app outside of Docker; version 16.18.0 is recommended)
+- Node.js (if running the app outside of Docker; version 18.20.4 is recommended)
 - Ports 1337, 5173 and 5432 should be free for the application if using default settings
   -  These ports can be changed in the `.env` file if desired.
 
 ## Getting started
 
-To build the development Docker image, go to the project root folder make a copy of the `.env.example` file and rename 
-the copy as `.env`.
+Backend module contains `strapi-plugin-import-export-entries` directoty which is a separate git repository. In order to initialise it you need to run:
 
-Then you need to install the git submodules by running `git submodule update --init --recursive`.
+```bash
+git submodule update --init --recursive
+```
 
-Finally, run `yarn install` and `yarn dev` in the project root folder.
-These commands will create a Docker image that has the frontend, the backend and a database in a single bundle.
+Then install dependencies for all workspaces:
+
+```bash
+yarn install
+```
+
```

Comment:
I agree on the hot-reloading, but I am not sure about the purpose of this docker setup guide otherwise. I mean does this document attempt to explain how to start the services using existing docker compose yaml files or does this document attempt to describe/explain what steps Dockerfiles contain within themselves? 

I would say that Dockerfiles should explain themselves (they are readable enough and if something is too complex we could leave a comment in there). So this document should explain how to start the services using docker compose files. In that case - we don't need to go into detail of what happens inside the Dockerfiles (which modules are being build and which dependencies are being installed), unless of course one needs to do something manually prior to docker compose execution. 

What do you think?

Reply:
Strictly speaking it's true that the note is not about Docker setup per se, but as this doc de facto serves as a "getting started with development" guide, it might be useful to mention that bit here as well. When reorganising the documentation later, we could separate the two.

(Just to make sure what I mean by IDE availaibility is that VS Code is able to link to the module. I guess another option would to just bundle that command into the `dev` script…)

Anyway, if you feel like adding this isn't kosher, we can leave it out and maybe add an item in troubleshooting?

Reply:
Right, I see your point on the IDE. I feel like these general notes belong to README at the root of the project. Which should explain the structure of the monorepo, dev tools setup and gotchas. I will try and do that now.

For the future, I would say that this document shall go in more detail on Docker specific things, f.e. that each module has it's own .env, compose yaml and Docker files. Explaining step by step how they can be started locally separately or altogether from the project's root with compose yaml. 🤔 

I would gladly implement my own vision if you give me the green light - I enjoy writing in general, but it could take me some time, not sure it has the priority now?

Reply:
Great, that sounds logical!

If you feel like, it'd be easiest to write the Docker docs now that everything's still fresh in the memory, go ahead and do it. Otherwise, let's move that to a separate issue for later.

Reply:
Alright, I updated the `README` in the root to describe the different module resolution mechanisms that take place within the monorepo. I will create a separate issue on updating the Docker docs. :)